crates/services/p2p/src/peer_manager.rs

```diff
@@ -291,22 +295,14 @@ impl PeerManager {
             .send(self.reserved_connected_peers.len());
     }
 
-    fn insert_peer_info(&mut self, peer_id: &PeerId, data: PeerInfoInsert) {
-        let peers = if self.reserved_peers.contains(peer_id) {
+    fn get_relevant_peers_mut(
```

Comment:
It's not clear what this is "relevant" for. Maybe something like this instead?

```suggestion
    fn get_peer_table_mut(
```